src/main/java/sirius/biz/storage/layer1/ObjectStorageSpace.java

```diff
@@ -103,6 +107,30 @@ public void upload(String objectId, InputStream inputStream, long contentLength)
         }
     }
 
+    /**
+     * Provides direct access to the contents of the requested object.
+     *
+     * @param objectId the physical storage key
+     * @return the contents a input stream
+     */
+    public Optional<InputStream> getInputStream(String objectId) {
+        try {
+            if (Strings.isEmpty(objectId)) {
+                return Optional.empty();
+            }
+            return Optional.ofNullable(engine.getAsStream(name, objectId));
+        } catch (IOException e) {
+            throw Exceptions.handle()
+                            .error(e)
+                            .to(StorageUtils.LOG)
+                            .withSystemErrorMessage(
```

Comment:
Parameters missing: 2 arguments x 4 %s placeholders

Reply:
The two last values are automagically added by withSystemErrorMessage where the first is the error message and the second is the name of the exception class